packages/new-editor/src/components/editor/variables/master/Variables.tsx

```diff
@@ -13,16 +13,20 @@ import {
 } from '@axonivy/ui-components';
 import { IvyIcons } from '@axonivy/ui-icons';
 import { flexRender, getCoreRowModel, useReactTable, type ColumnDef, type Row } from '@tanstack/react-table';
+import { useState } from 'react';
 import { Control } from '../../control/Control';
 import type { Variable } from '../../data/Variable';
 
 type VariablesProps = {
-  variables: Array<Variable>;
```

Comment:
I prefer Array<Variable> and also typesystem has better support for it...
https://www.totaltypescript.com/array-types-in-typescript